cmd/dequeuer/main.go

```diff
@@ -180,7 +180,10 @@ func main() {
 			TargetURL:  targetURL,
 		}
 
-		asyncStatsReporter := dequeuer.NewAsyncPrometheusStatsReporter()
+		asyncStatsReporter := dequeuer.NewAsyncPrometheusStatsReporter(awsClient.SQS(), queueURL, log)
+		stopStatsReporter := asyncStatsReporter.Start()
```

Comment:
Will this run on every async replica?

Reply:
that's a very good point, maybe it's not worth it to move this metric out of the operator?

Reply:
We can discuss this next week